app/tests/test.py

```diff
@@ -4,14 +4,14 @@
 import os
 
 is_travis = 'TRAVIS' in os.environ
-
+os.environ['MOZ_HEADLESS'] = '1'
 
 class BadgeyayTest(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         if is_travis:
-            cls.driver = webdriver.PhantomJS()
+            cls.driver = webdriver.Firefox()
         else:
             cls.driver = webdriver.Firefox()

```

Comment:
Remove the if-else condition as there is no need

Reply:
Yeah sure working on it